custom_components/powercalc/translations/de.json

```diff
@@ -0,0 +1,176 @@
+            "linear_mandatory": "Sie müssen mindestens Minimale Leistung oder Kalibrierungswert angeben",
+            "linear_min_higher_as_max": "Maximale Leistung kann nicht niedriger sein als Minimale Leistung"
+        },
+        "step": {
+            "daily_energy": {
+                "data": {
+                    "name": "Name",
+                    "on_time": "Dauer",
+                    "start_time": "Startzeit",
+                    "unique_id": "Unique-Id",
+                    "unit_of_measurement": "Maßeinheit (kWh oder W)",
+                    "update_frequency": "Aktualisierungsfrequenz",
+                    "value": "Wert",
+                    "value_template": "Wert-Template"
+                },
+                "data_description": {
+                    "on_time": "Wenn leer, ist der Standardwert 1 Tag, d.h. immer an",
+                    "update_frequency": "Zeit in Sekunden zwischen den Zustandsaktualisierungen des Sensors"
+                },
+                "title": "Einen daily fixed-Sensor erstellen"
+            },
+            "fixed": {
+                "data": {
+                    "power": "Leistung",
+                    "power_template": "Leistung-Template",
+                    "states_power": "Leistung je Zustand"
+                },
+                "data_description": {
+                    "power": "Ein fester Leistungswert in Watt, wenn die Einheit eingeschaltet ist",
+                    "power_template": "Template wird ausgewertet, um den Leistungswert in W zu erhalten",
+                    "states_power": "Ein 'state: power' Paar in jeder Zeile, siehe Beispiel oben"
+                },
+                "description": "Definieren Sie einen festen Leistungswert für Ihre Entität. Alternativ können Sie auch einen Leistungswert pro State definieren. Zum Beispiel:\n\n`playing: 8.3`\n`paused: 2.25`",
+                "title": "Fixed Konfiguration"
+            },
+            "group": {
+                "data": {
+                    "create_utility_meters": "Utilitymeter erstellen",
+                    "group_energy_entities": "Energie-Entitäten der Gruppe",
+                    "group_power_entities": "Leistungs-Entitäten der Gruppe",
+                    "name": "Name",
+                    "sub_groups": "Untergruppen",
+                    "unique_id": "Unique-Id"
+                },
+                "data_description": {
+                    "sub_groups": "Alle Sensoren aus den ausgewählten Untergruppen werden ebenfalls zu dieser Gruppe hinzugefügt"
+                },
+                "title": "Einen Gruppensensor erstellen"
+            },
+            "linear": {
+                "data": {
+                    "attribute": "Attribute",
+                    "calibrate": "Kalibrierungswert",
+                    "gamma_curve": "Gammakurve",
+                    "max_power": "Maximale Energie",

```

Comment:
This should be "Maximale Leistung" imho.

Reply:
Yes. Had this at the other line already, missed this. Corrected now.